uikit/src/main/res/layout/activity_bank_points.xml

```diff
@@ -31,6 +31,17 @@
             android:background="@color/dim_background"
             android:visibility="gone"/>
 
+        <com.midtrans.sdk.uikit.widgets.FancyButton
+            android:id="@+id/button_pay_without_point"
```

Comment:
Need to set the visibility to "GONE" until we fully support it.

Reply:
ah my false, it should be from this branch https://github.com/veritrans/veritrans-android/tree/chor/add-button-pay-without-point  . i'll remove it